### DISCUSSION TITLE
testrunner: how can i call global mocha hooks?

### DISCUSSION THREAD

**Discussion by blikblum** [modernweb-dev/web#2004] [General]
With mocha i can call mocha hooks to, for example, configure mocha-chai with a module that exports `mochaGlobalSetup` and call it with "require" command line param 

```js
import chai from 'chai'
import sinonChai from 'sinon-chai'

export async function mochaGlobalSetup() {
  chai.use(sinonChai)
}
```

package.json:

"scripts": {
    "test": "mocha --require test/setup.js test/*.spec.js",
    "test:watch": "mocha --watch --require test/setup.js test/*.spec.js"
  },

Is it possible to do with wtr?

**Comment by nidamunir-extreme** [comment]
Did you find a solution? I am seeing a typescript error in before()

**Reply by blikblum** [reply]
> Did you find a solution? I am seeing a typescript error in before()

No
